aspnetcore/tutorials/signalr.md

```diff
@@ -84,23 +84,21 @@ At the end, you'll have a working chat app:
 
 ## Add the SignalR client library
 
-The SignalR server library is included in the `Microsoft.AspNetCore.App` metapackage. The JavaScript client library isn't automatically included in the project. For this tutorial, you use Library Manager (LibMan) to get the client library from *unpkg*. unpkg is a content delivery network (CDN)) that can deliver anything found in npm, the Node.js package manager.
+The SignalR server library is included in the ASP.NET 3.0 shared framework. The JavaScript client library isn't automatically included in the project. For this tutorial, you use Library Manager (LibMan) to get the client library from *unpkg*. unpkg is a content delivery network (CDN)) that can deliver anything found in npm, the Node.js package manager.
 
 # [Visual Studio](#tab/visual-studio/)
 
 * In **Solution Explorer**, right-click the project, and select **Add** > **Client-Side Library**.
 
 * In the **Add Client-Side Library** dialog, for **Provider** select **unpkg**.
 
-* For **Library**, enter `@aspnet/signalr@1`, and select the latest version that isn't preview.
-
-  ![Add Client-Side Library dialog - select library](signalr/_static/libman1.png)
+* For **Library**, enter `@aspnet/signalr@3.0.0-preview6.19307.2`.
```

Comment:
FYI, this will be changing to `@microsoft/signalr` in preview7

Reply:
> Is including aspnetcore/tutorials/signalr/sample/wwwroot/lib/signalr/dist/browser/signalr.js necessary?

The tutorial currently includes a sample project that you can download and run as-is.  We could eliminate that, and only include the code files that we use for displaying in the tutorial.  Now that you mention it, I think that makes sense for a simple tutorial like this -- I'll go ahead and eliminate the downloadable project.  

Note, however, that this is a Draft PR because it's incomplete. I haven't had a chance to fix issues that appear in staging and haven't updated the VS Code and VS Mac instructions yet. 

I'm aware of the package change and will make those changes in the PR or in master depending on when the PR is merged.